src/main/java/se/michaelthelin/spotify/SpotifyHttpManager.java

```diff
@@ -91,13 +91,15 @@ public SpotifyHttpManager(Builder builder) {
       .custom()
       .setDefaultCredentialsProvider(credentialsProvider)
       .setDefaultRequestConfig(requestConfig)
+      .disableAutomaticRetries()
```

Comment:
Hmm, I would've hoped that automatic retries are disabled for status code 429 requests by default :smile: Do you think disabling automatic retries is required? Do you have an idea how to disable only code 429 requests or is it necessary to disable all automatic retries? :raised_hands:

Reply:
@SergioBarbero what's the status on this?

Reply:
> Hmm, I would've hoped that automatic retries are disabled for status code 429 requests by default 😄 Do you think disabling automatic retries is required? Do you have an idea how to disable only code 429 requests or is it necessary to disable all automatic retries? 🙌

see [HttpRequestRetryStrategy.java](https://github.com/apache/httpcomponents-client/blob/master/httpclient5/src/main/java/org/apache/hc/client5/http/HttpRequestRetryStrategy.java) and implementation [DefaultHttpRequestRetryStrategy.java](https://github.com/apache/httpcomponents-client/blob/master/httpclient5/src/main/java/org/apache/hc/client5/http/impl/DefaultHttpRequestRetryStrategy.java). You would need to create a new implementation and set strategy retry on http client.
In my opinion, you should just copy most of the implementation from `HttpRequestRetryStrategy`, just make so it doesn't retry on 429.

Reply:
I've implemented your recommendation, could you please check @hwup?